yt_dlp/extractor/moviepilot.py

```diff
@@ -91,22 +86,21 @@ def _real_extract(self, url):
 
         webpage = self._download_webpage(f'https://www.moviepilot.de/movies/{video_id}/trailer', video_id)
 
-        duration = try_get(
-            re.match(r'P(?P<hours>\d+)H(?P<mins>\d+)M(?P<secs>\d+)S',
-                     self._html_search_meta('duration', webpage, fatal=False) or ''),
-            lambda mobj: sum(float(x) * y for x, y in zip(mobj.groups(), (3600, 60, 1))))
-        # _html_search_meta is not used since we don't want name=description to match
-        description = self._html_search_regex(
-            '<meta[^>]+itemprop="description"[^>]+content="([^>"]+)"', webpage, 'description', fatal=False)
+        clip_store = self._search_json(
+            r'<script id="__NEXT_DATA__" type="application/json">',
+            webpage, 'clip store', video_id, fatal=True)
+        clip = traverse_obj(clip_store, ('props', 'initialProps', 'pageProps'),
+                            expected_type=dict) or {}
```

Comment:
It doesnt make sense for this to be non-fatal

Reply:
I have replaced it now with a direct dict access and explicit error message. Should I further use `traverse_obj`?